doc/content/getting-started/programming-software/st-examples/stcubeprog/_index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "STM32CubeProgrammer"
+description: ""
+weight:
+distributions: null
+---
+
+[STM32CubeProgrammer](https://www.st.com/en/development-tools/stm32cubeprog.html) is the easiest option that allows for multiple configuration options to work with any ST-LINK. It has both a graphical and a command-line interface. It has support for multiple operating systems. 
+
+<!--more-->
+
+## GUI version
+
+1. Launch STM32CubeProgrammer.
+
+{{< figure src="cubeprog_startupscreen.png" alt="cubeprog startup screen" >}}
+
+2. Choose your preferred connection settings and click on **Connect** on the top right.
+
+{{< figure src="cubeprog_programming_button.png" alt="cubeprog connect button" >}}
+
+3. Choose the ***Erasing and Programming*** icon on the left.
+
+{{< figure src="cubeprog_options.png" alt="cubeprog erase option" >}}
+
+4. In the download section, choose your executable file location and start address (0x08000000). Then hit the ***Start Programming*** button.
```

Comment:
```suggestion
4. In the download section, choose the location of your executable file and start address (`0x08000000`), then hit the **Start Programming** button.
```

Reply:
what is this 0x08000000 value? is it something default or something you're showing as an example?

Reply:
Default start address of the application, basically the chip will first check your boot mode and if its set to running the user software then it will jump to this memory address. This starts the application. It is the default in all ST chips I know of (and in the STM32WL chip as well).

Reply:
ok, I edited the suggestion just to note that it's default